<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Trantor-LP260" failures="1" tests="5" name="TestRunner.loginTestRunner" time="29.188" errors="0" timestamp="2024-08-09T18:10:04 IST" skipped="0">
  <testcase classname="TestRunner.loginTestRunner" name="runScenario" time="5.145"/>
  <testcase classname="TestRunner.loginTestRunner" name="runScenario" time="3.707"/>
  <testcase classname="TestRunner.loginTestRunner" name="runScenario" time="7.878"/>
  <testcase classname="TestRunner.loginTestRunner" name="runScenario" time="7.263"/>
  <testcase classname="TestRunner.loginTestRunner" name="runScenario" time="5.195">
    <failure message="expected [Epic sadface: Username and password do not match any user in this service] but found [Epic sadface: Username and password do not match any user in this servicee]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Epic sadface: Username and password do not match any user in this service] but found [Epic sadface: Username and password do not match any user in this servicee]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at PageFactory.loginPageFactory.loginErrorMsg(loginPageFactory.java:94)
at StepDefinitions.loginStepDefinitions.should_show_the_error_message(loginStepDefinitions.java:62)
at âœ½.Should show the error message(file:///C:/Users/somendra.chauhan/eclipse-workspace/BDDProject/src/test/resources/FeatureFile/login.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- TestRunner.loginTestRunner -->
